src/utils/pureFunctions.ts

```diff
@@ -11,6 +11,14 @@ export const shortenString = (
   return start + separator + end;
 };
 
+export const truncateString = (string: string, toLength = 13): string => {
+  if (string.length > toLength) {
+    return string.slice(0, toLength) + '...';
+  } else {
+    return string;
+  }
+};
```

Comment:
```suggestion
export const truncateString = (
  string: string | null,
  toLength = 13
): string => {
  if (string && string.length > toLength) {
    return string.slice(0, toLength) + '...';
  } else {
    return string!;
  }
};
```

https://github.com/bancorprotocol/webapp-v2/pull/489#discussion_r773804500

Reply:
/cc @zavelevsky